envergo/hedges/services.py

```diff
@@ -0,0 +1,124 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import TYPE_CHECKING, Literal
+
+from envergo.evaluations.models import RESULTS
+from envergo.hedges.models import HedgeData
+
+if TYPE_CHECKING:
+    from envergo.moulinette.models import MoulinetteHaie
+
+
+class PlantationResults(Enum):
+    Adequate = "adequate"
+    Inadequate = "inadequate"
+
+
+class PlantationEvaluator:
+
+    def __init__(self, moulinette: "MoulinetteHaie", hedge_data: HedgeData):
+        self.moulinette = moulinette
+        self.hedge_data = hedge_data
+        self.evaluate()
+
+    @dataclass
+    class EvaluationResult:
+        result: Literal[PlantationResults.Adequate, PlantationResults.Inadequate]
+        conditions: list[str]
+
+    @property
+    def result(self):
+        """Return the evaluator result.
+
+        This value is used to rendered the plantation result templates.
```

Comment:
Je propose cette suggestion, ça te paraît pertinent ?

```suggestion
        This value is used to select the plantation result templates.
```